app/views/searches/results.csv.erb

```diff
@@ -1,4 +1,4 @@
-"Name","Account Number","Year Carried Over","Credit Details","Date Account Opened"
+"Name","Account Number","Year Carried Over","Credit Details","Date Account Opened","Case Link"
 <% @cases.each do |kase| %>
-<%= raw kase.case_name %>,<%= kase.account_number %>,<%= kase.year_carried_over %>,<%= raw kase.credit_details %>,<%= kase.date_account_opened %>
+<%= raw kase.case_name %>,<%= kase.account_number %>,<%= kase.year_carried_over %>,<%= raw kase.credit_details %>,<%= kase.date_account_opened %>,<%= "#{(case_url(kase.account_number))}" %>
```

Comment:
You don't need the string interpolation stuff (`#{}`) here as it's just rendering a variable so `<%= case_url(kase.account_number) %>` will work fine